src/transforms/push.ts

```diff
@@ -12,20 +12,23 @@ import { JobOptions } from "bull";
 
 const logger = getLogger("transforms.push");
 
-function mapFile(githubFile) {
+const mapFile = (repoName: string, repoOwner: string, commitHash: string) => (githubFile) => {
```

Comment:
Can you explain this construct?

Reply:
Instead of adding complexity of a function returning another function, just create a flat function with all the parameters and use an arrow function below: `filesToSend.map(file => mapFile(file, repo, owner.name, sha.id))`